source/renderer/app/components/notifications/NotificationActions.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from 'react';
+import classnames from 'classnames';
+import { map } from 'lodash';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { Link } from 'react-polymorph/lib/components/Link';
+import { observer } from 'mobx-react';
+import styles from './NotificationActions.scss';
+
+export type NotificationActionItems = Array<NotificationActionItem>;
+
+export type NotificationActionItem = {
+  className?: ?string,
+  label: string | Node,
+  primary?: boolean,
+  disabled?: boolean,
+  onClick?: Function,
+  onDisabled?: Function,
+  isLink?: boolean,
+  hasIconAfter?: boolean,
+  hasIconBefore?: boolean,
+  autoFocus?: boolean,
+};
+
+type Props = {
+  actions: NotificationActionItems,
+};
+
+const NotificationActions = observer(({ actions }: Props) => {
+  const componentStyles = classnames([styles.component]);
+  return (
+    <div className={componentStyles}>
+      {map(actions, (action, key) => {
```

Comment:
Maybe destructuring in line 33 saves you code?  like: "{map(actions, ({className, primary, autoFocus, label, onClick, disabled ....}, key) => {

Reply:
Done